lib/controls/ValueSelector.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class ValueSelector extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {value, options, className, handleOnChange} = this.props;
+    const children = this.props.children;
+
+    if (children) {
+      return (
+        React.cloneElement(children,
```

Comment:
Or are you saying setup the defaults within the `QueryBuilder` component and then allow users to override it with props? I like that better actually